packages/discovery-provider/plugins/pedalboard/apps/relay/src/middleware/antiAbuse.ts

```diff
@@ -42,27 +49,20 @@ export const antiAbuseMiddleware = async (
     logger.error({ error: e }, 'AAO uncaught exception')
   })
 
-  if (!user.handle) {
-    logger.error({ validatedRelayRequest }, "user found without handle")
-    internalError(
-      next,
-      `user found but without handle, investigate ${JSON.stringify(user)}`
-    )
-    return
-  }
-
-  // read from cache and determine if blocked from relay
-  const userAbuseRules = await readAAOState(user.handle)
-  if (userAbuseRules === null) {
-    // first relay, allow passage
-    next()
-    return
-  }
+  if (user.handle) {
```

Comment:
can we check if user is also a guest? we may want to have coverage for the case where they dont have handle and are not guest?

Reply:
i think the only way to truly verify this is a guest is check identity and i'm hesitant to add that dependency. i think there might be valid cases for not passing a handle. im gonna switch handle to user ID